runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkExecutionEnvironments.java

```diff
@@ -375,4 +377,10 @@ private static void setManagedMemoryByFraction(final Configuration config) {
       config.setString("taskmanager.memory.managed.size", String.valueOf(managedMemorySize));
     }
   }
+
+  private static void disableClassLoaderLeakCheck(final Configuration config) {
```

Comment:
Just for pure curiosity, why is this needed?

Reply:
I believe this is because of GRPC which may attempt to load classes after the Flink job has been removed. Loading new classes on closed classloaders will error without this setting.

Reply:
In any case, we should add a note or JIRA ticket to why this is necessary.

Reply:
Done.